app/gui/integration-test/project-view/componentBrowser.spec.ts

```diff
@@ -100,14 +100,14 @@ test('Opening Component Browser from output port buttons', async ({ page }) => {
   const createNodeFromPortButton = await locate.createNodeFromPortButton(page, node)
   await expect(createNodeFromPortButton).toBeVisible()
   await createNodeFromPortButton.click({ force: true })
-  await expectAndCancelBrowser(page, '', null, 'selected')
+  await expectAndCancelBrowser(page, '', 'Input Components', 'selected')
```

Comment:
The tests were right: we don't want to display `'Input Components'` with source node.

Reply:
Thanks. I was a bit confused by the fact that tests differ from the actual IDE behavior. The reason is that in tests, we don’t have expression info available on some of the nodes, so no label is indeed visible. In practice, though, we have type information for all nodes, so the label is visible almost always.